src/controllers/expenses.controllers.js

```diff
@@ -0,0 +1,103 @@
+'use strict';
+
+const expensesService = require('../services/expenses.service');
+
+const get = async(req, res) => {
+  const { userId, categories, from, to } = req.body;
+
+  const expenses = expensesService.getExpenses({
+    userId, categories, from, to,
+  });
+
+  if (!expenses) {
+    res.sendStatus(404);
+
+    return;
+  }
+  res.send((await expenses).map(expense => expensesService.normalize(expense)));
+};
+
+const getOne = (req, res) => {
+  const { id } = req.params;
+  const expense = expensesService.getExpenseById(id);
+
+  if (!expense) {
+    res.sendStatus(404);
+
+    return;
+  }
+  res.send(expensesService.normalize(expense));
+};
+
+const remove = (req, res) => {
+  const { id } = req.params;
+
+  const expense = expensesService.getExpenseById(id);
+
+  if (!expense) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  expensesService.deleteExpense(expense).then(() => {
+    // eslint-disable-next-line no-console
```

Comment:
you can disable console for whole file to avoid using this rule couple more times in the code